src/views/Proposal.vue

```diff
@@ -168,7 +168,7 @@
           :results="results"
         />
         <PluginGnosisCustomBlock
-          v-if="_get(payload, 'metadata.plugins.gnosis.baseTokenAddress')"
+          v-if="payload.metadata.plugins?.gnosis.baseTokenAddress"
```

Comment:
It should handle the case where `plugins` exist but not `gnosis`
```suggestion
          v-if="payload.metadata.plugins?.gnosis?.baseTokenAddress"
```